src/main/java/Task.java

```diff
@@ -0,0 +1,16 @@
+public abstract class Task {
+    protected Boolean done;
+    protected String taskContent;
+
+    public Task(String input) {
+        this.taskContent = input;
+        this.done = false;
+    }
+    public void mark() {
```

Comment:
Consider renaming the name of the method to `markAsDone` to make the function more descriptive. Anyways this was the example provided in Level-3.